video_xblock/mixins.py

```diff
@@ -623,6 +623,5 @@ def usage_id(self):
 
         Returns stub value if `location` property is unavailabe. E.g. in workbench runtime.
         """
-        if hasattr(self, 'location'):
-            return self.location.to_deprecated_string()
-        return 'usage_id'
+        # TODO: Replace usage_id property to block_id, to_deprecated_string does not exists furthermore.
```

Comment:
Did the issue for that technical debt was added? If yes could you please provide a link to it, otherwise let's add a new one to ensure we won't forget it.

Reply:
https://github.com/raccoongang/xblock-video/issues/415